src/App.js

```diff
@@ -42,15 +43,14 @@ function App() {
   const submitAnswer = (answer) => {
     answer = answer.trim()
     if (currentAnswers.includes(answer)) {
-      console.log(`${answer} has already been guessed!`)
+      setErrors([...errors, `${answer} has already been guessed!`])
```

Comment:
Why do we store a list of all the errors, instead of storing just the latest error?

Reply:
No particular reason, actually. I'll take it out. Still looking for whatever is happening between local and deployed. I'm stumped. Versions are the same. The class gets applied. Got any ideas?